docs/release-notes/2.40.1.md

```diff
@@ -11,18 +11,25 @@ Percona Monitoring and Management (PMM) is an open-source database monitoring, m
 - [PMM-12592](https://jira.percona.com/browse/PMM-12592) - Fixed an issue where the **Library Panels** for the PMM dashboard were not working after upgrade to PMM 2.40.0.
 - [PMM-12576](https://jira.percona.com/browse/PMM-12576) - After upgrading to PMM 2.40.0, changing the Admin user's password from the terminal was not functioning. The issue has been resolved now.
 - [PMM-12587](https://jira.percona.com/browse/PMM-12587) - After upgrading to PMM 2.40.0, some users may experience incorrect mappings between dashboards, folders, users, and groups. This can result in either a successful upgrade or a `500 internal server error`. The issue has now been resolved.
-- [PMM-12590](https://jira.percona.com/browse/PMM-12590) [CVE-2023-4911](https://nvd.nist.gov/vuln/detail/CVE-2023-4911) is a vulnerability in the base OS that PMM is based on. It has been fixed in the base OS, and the fix is available in PMM.
+- [PMM-12590](https://jira.percona.com/browse/PMM-12590) [CVE-2023-4911](https://nvd.nist.gov/vuln/detail/CVE-2023-4911) is a vulnerability in the OS that PMM is based on. It has been fixed in the base OS, and the fix is available in PMM.
 
 
 ## How to upgrade to PMM 2.40.1
 
-For instructions on upgrading to PMM 2.40.1 and mitigating issues from PMM 2.40.0, follow the steps below:
+!!! caution alert alert-warning "Important"
+    The steps below are only applicable to users who have upgraded to 2.40.0 and have been experiencing this problem. 
+
+To upgrade to PMM 2.40.1:
 
 1. Copy the file from `/srv/backup/grafana/grafana.db` to `/srv/grafana/grafana.db`
 
+    !!! caution alert alert-warning "Caution"
+        This step will cause any changes made to Grafana after upgrading to version 2.40.0 to be lost.
```

Comment:
`This step will cause a loss of all changes made to Grafana after upgrading to version 2.40.0.`
Please consider this one, at least for me it "sound" better.